src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -58,28 +58,33 @@ Amber::Server.configure do |settings|
   #
   # > Read more about Linux PORT REUSE https://lwn.net/Articles/542629/
   #
-  # settings.port_reuse = ENV["PORT_REUSE"]
+  # settings.port_reuse = true
   #
   #
   # Process Count: This will enable Amber to be used in cluster mode,
   # spinning an instance for each number of process specified here.
   # Rule of thumb, always leave at least 1 core available for system processes/resources.
   #
-  # settings.port = (ENV["PORT"] ||= 80).to_i
+  # settings.process_count = ENV["PROCESS_COUNT"].to_i if ENV["PROCESS_COUNT"]?
```

Comment:
Should there be a hardcoded fallback here?
```crystal
settings.process_count = ENV["PROCESS_COUNT"]? ? ENV["PROCESS_COUNT"].to_i : 1 
```

Reply:
We already have `process_count = 1` [here](https://github.com/amberframework/amber/blob/c7ffd11d6c65c5347aca41fa0580c7c8d6265e60/src/amber/server/settings.cr#L27)

Notice `]?` , so returns `nil` if not found and setting attribute is not assigned.

Reply:
Yes. There is already a fall back for all of them. They just weren't being used if the environment var was missing.